huggingface/pytorch/inference/docker/1.9/py3/sdk1.16.1/Dockerfile.neuron

```diff
@@ -0,0 +1,139 @@
+FROM ubuntu:20.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+# Specify accept-bind-to-port LABEL for inference pipelines to use SAGEMAKER_BIND_TO_PORT
+# https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+ARG MMS_VERSION=1.1.5
```

Comment:
@philschmid do we need MMS for this release?

Reply:
What do you mean if we need it?